app/_hub/kong-inc/ai-semantic-prompt-guard/overview/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+nav_title: Overview
+---
+
+The AI Semantic Prompt Guard plugin is an enhancement of the AI Prompt Guard plugin; it lets you to allow or block prompts based on a list of similar prompts, to guard against misuse of `llm/v1/chat` or `llm/v1/completions` requests.
```

Comment:
```suggestion
The AI Semantic Prompt Guard plugin enhances the AI Prompt Guard plugin by allowing you to permit or block prompts based on a list of similar prompts, helping to prevent misuse of `llm/v1/chat` or `llm/v1/completions` requests.
```